### DISCUSSION TITLE
Change memory restriction on unraid

### DISCUSSION THREAD

**Discussion by pastrygator** [wolveix/satisfactory-server#257] [Q&A] [answered]
Hi, sorry I feel like an idiot but I can't seem to wrap my head around this. I have this container installed on/through Unraid. The documentation makes it look like the upper reservation on memory is 6G by default. In anticipation of the 1.0 release I'm trying to host the server for me and a couple friends, and while our fresh world is only using a little over a gig of RAM, I'm sure that will start to creep up as time goes on.

On the docker container page in unraid it shows the memory available for the container as out of what is available to all the containers (125 GiB). Will it actually go beyond the 6G? or do I need to change that in the container's configuration? HOW do I change that? I can't find anything anywhere in the settings of the container for me to change or edit the upper reservation/amount of memory on Unraid.

Thanks in advance for the help. Still really green with docker/containers/etc

**Comment by wolveix** [comment, marked as answer]
Hey! I'm unfamiliar with Unraid specifically; however, if you're not enforcing a restriction already (and it doesn't sound like you are), then it'll consume memory from your total available without restriction :)

**Reply by pastrygator** [reply]
Thanks so much for the reply! I'll report back once we inevitably eventually start creeping up on the RAM usage and see if that's the case. I had kind of figured that was probably the answer, but I just wanted to be proactive and make sure I wasn't missing something super obvious. Thanks again, and for your work!
